### PR TITLE
Carmen Sandiego

### PR DESCRIPTION
**Pull request by fugimax**
Now with more map.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,14 +10,15 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div id="header">
-      The HITMAN Hat Game
-    </div>
+    <div id="header"><img src=hitman_logo.jpg width=500></div>
     <div id="content">
+        Pick your stage, targets, weapon, and disguise.</p>
```
Comment (alanjoyce) [inline index.html:15]: There's an ending /p here but no starting p tag. Consider adding a starting tag.

File: index.html [hunk 2]
```diff
@@ -10,14 +10,15 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <div id="header">
-      The HITMAN Hat Game
-    </div>
+    <div id="header"><img src=hitman_logo.jpg width=500></div>
```
Comment (alanjoyce) [inline index.html:13]: Should put quotes around the filename.

File: index.html [hunk 3]
```diff
@@ -10,14 +10,15 @@
...
     <div id="content">
+        Pick your stage, targets, weapon, and disguise.</p>
+        <p>Feeling dangerous? Pick a wildcard.</p>
+      <p></p>
```
Comment (alanjoyce) [inline index.html:17]: Is this paragraph just here for spacing? If so, consider doing the spacing in CSS.

**Review by alanjoyce — CHANGES_REQUESTED** [review]
A few comments.

File: index.html (hunk 3 above)
Comment (fugimax) [reply index.html:17]: Nah, i'm good

**Pull request closed without merging**